configs/qcom_sdm636_bbry_luna_defconfig

```diff
@@ -11,6 +11,10 @@ CONFIG_DEFAULT_DEVICE_TREE="qcom/sdm636-bbry-luna-boe"
 CONFIG_REQUIRE_SERIAL_CONSOLE=n
 CONFIG_SERIAL_PRESENT=n
 CONFIG_NO_FB_CLEAR=n
+CONFIG_DEBUG_UART=n
+CONFIG_DEBUG_UART_MSM=n
+CONFIG_DEBUG_UART_ANNOUNCE=n
+CONFIG_MSM_SERIAL=n
```

Comment:
So many disables, looks like overkill. I think disabling only DEBUG_UART should be sufficient, then DEBUG_UART_MSM & DEBUG_UART_ANNOUNCE should disappear from the resulting `.config`.

Can you try keeping CONFIG_MSM_SERIAL enabled..?

Reply:
Ok I'll try only debug_uart and double check the devicetree later